2019-04-28 09:12:32.135131:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 2 sec before restart the streaming.



2019-04-28 09:12:34.137615:
restarting the stream...

2019-04-28 09:12:34.208285:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980cf8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 2 sec before restart the streaming.



2019-04-28 09:12:36.211073:
restarting the stream...

2019-04-28 09:12:36.217529:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8695588>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 4 sec before restart the streaming.



2019-04-28 09:12:40.218646:
restarting the stream...

2019-04-28 09:12:40.225150:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 8 sec before restart the streaming.



2019-04-28 09:12:48.233543:
restarting the stream...

2019-04-28 09:12:48.242511:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86955f8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 16 sec before restart the streaming.



2019-04-28 09:13:04.259274:
restarting the stream...

2019-04-28 09:13:04.265633:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa11c978>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 32 sec before restart the streaming.



2019-04-28 09:13:36.294149:
restarting the stream...

2019-04-28 09:13:36.296822:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa125358>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 64 sec before restart the streaming.



2019-04-28 09:14:40.361441:
restarting the stream...

2019-04-28 09:14:40.368180:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa125ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 128 sec before restart the streaming.



2019-04-28 09:16:48.462098:
restarting the stream...

2019-04-28 09:16:48.466491:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980f60>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 256 sec before restart the streaming.



2019-04-28 09:21:04.566003:
restarting the stream...

2019-04-28 09:21:04.572333:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8695ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 512 sec before restart the streaming.



2019-04-28 09:29:36.672956:
restarting the stream...

2019-04-28 09:29:36.677779:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa11c358>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 09:45:36.774593:
restarting the stream...

2019-04-28 09:45:36.780167:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa11c198>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 10:01:36.880679:
restarting the stream...

2019-04-28 10:01:36.887177:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86951d0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 10:17:36.987842:
restarting the stream...

2019-04-28 10:17:36.994233:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980f60>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 10:33:37.087152:
restarting the stream...

2019-04-28 10:45:35.791454:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 960 sec before restart the streaming.



2019-04-28 11:01:35.890272:
restarting the stream...

2019-04-28 11:01:35.897052:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980c18>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 2 sec before restart the streaming.



2019-04-28 11:01:37.899796:
restarting the stream...

2019-04-28 11:01:37.908880:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115358>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 4 sec before restart the streaming.



2019-04-28 11:01:41.913816:
restarting the stream...

2019-04-28 11:01:41.920195:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3208>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 8 sec before restart the streaming.



2019-04-28 11:01:49.928795:
restarting the stream...

2019-04-28 11:01:49.934565:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115fd0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 16 sec before restart the streaming.



2019-04-28 11:02:05.950642:
restarting the stream...

2019-04-28 11:02:05.957234:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa1153c8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 32 sec before restart the streaming.



2019-04-28 11:02:37.989889:
restarting the stream...

2019-04-28 11:02:37.999132:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980e80>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 64 sec before restart the streaming.



2019-04-28 11:03:42.021630:
restarting the stream...

2019-04-28 11:03:42.028559:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3400>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 128 sec before restart the streaming.



2019-04-28 11:05:50.128977:
restarting the stream...

2019-04-28 11:05:50.138666:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980470>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 256 sec before restart the streaming.



2019-04-28 11:10:06.194196:
restarting the stream...

2019-04-28 11:10:06.200582:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3c50>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 512 sec before restart the streaming.



2019-04-28 11:18:38.298567:
restarting the stream...

2019-04-28 11:18:38.304096:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115470>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 11:34:38.402650:
restarting the stream...

2019-04-28 11:34:38.411714:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86bc710>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 11:50:38.510653:
restarting the stream...

2019-04-28 11:50:38.520881:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8695390>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-28 12:06:38.601808:
restarting the stream...

2019-04-29 19:08:54.787362:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 960 sec before restart the streaming.



2019-04-29 19:24:54.885954:
restarting the stream...

2019-04-29 19:54:39.856133:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 2 sec before restart the streaming.



2019-04-29 19:54:41.858965:
restarting the stream...

2019-04-29 19:54:41.865281:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa11ce48>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 2 sec before restart the streaming.



2019-04-29 19:54:43.868049:
restarting the stream...

2019-04-29 19:54:43.873615:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa125e10>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 4 sec before restart the streaming.



2019-04-29 19:54:47.878429:
restarting the stream...

2019-04-29 19:54:47.887915:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115dd8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 8 sec before restart the streaming.



2019-04-29 19:54:55.896716:
restarting the stream...

2019-04-29 19:54:55.905565:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980a90>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 16 sec before restart the streaming.



2019-04-29 19:55:11.922482:
restarting the stream...

2019-04-29 19:55:11.927214:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a39e8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 32 sec before restart the streaming.



2019-04-29 19:55:43.930454:
restarting the stream...

2019-04-29 19:55:43.936776:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8695d68>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 64 sec before restart the streaming.



2019-04-29 19:56:48.001548:
restarting the stream...

2019-04-29 19:56:48.011241:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8695ef0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 128 sec before restart the streaming.



2019-04-29 19:58:56.086473:
restarting the stream...

2019-04-29 19:58:56.095276:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3a58>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 256 sec before restart the streaming.



2019-04-29 20:03:12.166905:
restarting the stream...

2019-04-29 20:10:38.663673:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 512 sec before restart the streaming.



2019-04-29 20:19:10.764611:
restarting the stream...

2019-04-30 01:25:40.036096:
ProtocolError('Connection broken: IncompleteRead(2 bytes read)', IncompleteRead(2 bytes read))
waiting 2 sec before restart the streaming.



2019-04-30 01:25:42.105459:
restarting the stream...

2019-04-30 03:56:44.872026:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 2 sec before restart the streaming.



2019-04-30 03:56:46.927139:
restarting the stream...

2019-04-30 03:56:57.044490:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a35f8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 2 sec before restart the streaming.



2019-04-30 03:56:59.047102:
restarting the stream...

2019-04-30 03:57:14.107938:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3fd0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 4 sec before restart the streaming.



2019-04-30 03:57:18.110962:
restarting the stream...

2019-04-30 03:58:35.188934:
SSLError(MaxRetryError('HTTPSConnectionPool(host=\'stream.twitter.com\', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by SSLError(SSLError("bad handshake: SysCallError(-1, \'Unexpected EOF\')",),))',),)
waiting 8 sec before restart the streaming.



2019-04-30 03:58:43.195282:
restarting the stream...

2019-04-30 07:47:31.947939:
ProtocolError('Connection broken: IncompleteRead(2 bytes read)', IncompleteRead(2 bytes read))
waiting 16 sec before restart the streaming.



2019-04-30 07:47:47.989773:
restarting the stream...

2019-04-30 07:57:26.829212:
ProtocolError('Connection broken: IncompleteRead(2 bytes read)', IncompleteRead(2 bytes read))
waiting 2 sec before restart the streaming.



2019-04-30 07:57:28.832058:
restarting the stream...

2019-04-30 08:18:58.036914:
ProtocolError('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
waiting 2 sec before restart the streaming.



2019-04-30 08:19:00.039405:
restarting the stream...

2019-04-30 08:56:15.966268:
ProtocolError('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
waiting 2 sec before restart the streaming.



2019-04-30 08:56:17.985701:
restarting the stream...

2019-04-30 09:19:42.206056:
ProtocolError('Connection broken: IncompleteRead(0 bytes read)', IncompleteRead(0 bytes read))
waiting 2 sec before restart the streaming.



2019-04-30 09:19:44.208577:
restarting the stream...

2019-04-30 09:39:49.534930:
ReadTimeoutError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Read timed out.",)
waiting 2 sec before restart the streaming.



2019-04-30 09:39:51.722168:
restarting the stream...

2019-04-30 09:40:01.738901:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a38358>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 2 sec before restart the streaming.



2019-04-30 09:40:03.741433:
restarting the stream...

2019-04-30 09:40:13.754737:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86bc898>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 4 sec before restart the streaming.



2019-04-30 09:40:17.757957:
restarting the stream...

2019-04-30 09:40:27.766916:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115668>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 8 sec before restart the streaming.



2019-04-30 09:40:35.773753:
restarting the stream...

2019-04-30 09:40:45.792073:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 16 sec before restart the streaming.



2019-04-30 09:41:01.805986:
restarting the stream...

2019-04-30 09:41:11.822512:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115be0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 32 sec before restart the streaming.



2019-04-30 09:41:43.853750:
restarting the stream...

2019-04-30 09:41:53.866622:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a38f98>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 64 sec before restart the streaming.



2019-04-30 09:42:57.931458:
restarting the stream...

2019-04-30 09:43:07.948219:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa95dc18>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 128 sec before restart the streaming.



2019-04-30 09:45:16.045994:
restarting the stream...

2019-04-30 09:45:26.062690:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa95deb8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 256 sec before restart the streaming.



2019-04-30 09:49:42.161758:
restarting the stream...

2019-04-30 09:49:52.178720:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980ba8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 512 sec before restart the streaming.



2019-04-30 09:58:24.279263:
restarting the stream...

2019-04-30 09:58:34.294896:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86bc1d0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 10:14:34.393983:
restarting the stream...

2019-04-30 10:14:44.410735:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3d7b8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 10:30:44.510004:
restarting the stream...

2019-04-30 10:30:54.526679:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3d198>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 10:46:54.625686:
restarting the stream...

2019-04-30 10:47:04.642650:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a38b70>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 11:03:04.741800:
restarting the stream...

2019-04-30 11:03:14.758385:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980390>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 11:19:14.858055:
restarting the stream...

2019-04-30 11:19:24.875236:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a5f2e8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 11:35:24.962011:
restarting the stream...

2019-04-30 11:35:35.098402:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa95d898>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 11:51:35.199795:
restarting the stream...

2019-04-30 11:51:45.225643:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3df60>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 12:07:45.325751:
restarting the stream...

2019-04-30 12:07:55.339063:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980d30>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 12:23:55.439541:
restarting the stream...

2019-04-30 12:24:05.456364:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115550>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 12:40:05.557087:
restarting the stream...

2019-04-30 12:40:15.574754:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980828>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 12:56:15.673734:
restarting the stream...

2019-04-30 12:56:25.691312:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3d898>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 13:12:25.789961:
restarting the stream...

2019-04-30 13:12:35.802448:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3dba8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 13:28:35.902380:
restarting the stream...

2019-04-30 13:28:45.942695:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa1157b8>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 13:44:46.039052:
restarting the stream...

2019-04-30 13:44:56.054793:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa115e10>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 14:00:56.107984:
restarting the stream...

2019-04-30 14:01:06.122755:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3d860>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 14:17:06.221943:
restarting the stream...

2019-04-30 14:17:16.238521:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980080>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 14:33:16.337894:
restarting the stream...

2019-04-30 14:33:26.365595:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86bc748>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 14:49:26.435137:
restarting the stream...

2019-04-30 14:49:36.450616:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86bc240>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 15:05:36.549988:
restarting the stream...

2019-04-30 15:05:46.566597:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6afa980550>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 15:21:46.666027:
restarting the stream...

2019-04-30 15:21:56.682756:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a3dda0>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 15:37:56.781849:
restarting the stream...

2019-04-30 15:38:06.798293:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af86a3d68>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



2019-04-30 15:54:06.882953:
restarting the stream...

2019-04-30 15:54:16.911341:
ConnectionError(MaxRetryError("HTTPSConnectionPool(host='stream.twitter.com', port=443): Max retries exceeded with url: /1.1/statuses/filter.json?delimited=length (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f6af8a38588>: Failed to establish a new connection: [Errno -2] Name or service not known',))",),)
waiting 960 sec before restart the streaming.



